modules/understanding-insights-operator-alerts.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+
+:_content-type: CONCEPT
+[id="understanding-insights-operator-alerts_{context}"]
+= Understanding Insights Operator alerts
+
+Insights Operator sends alerts through the Prometheus monitoring system to Alertmanager. These alerts can be viewed in the Alerting UI which is accessible through the Administrator perspective and the Developer perspective in the {product-title} web console. 
```

Comment:
I am not sure that `sends` is the right word here TBH. Perhaps defines or declares?